.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### What does this PR do?
+
+
+
+
+
+#### Description of Task to be completed?
```

Comment:
Difference between this and above heading a little hard to tease out

Reply:
Thanks for pointing that. I was of the idea that "Description" would be a general a series of steps taken in fulfillment of the Pull Request.